python/orca/src/bigdl/orca/learn/pytorch/callbacks.py

```diff
@@ -0,0 +1,57 @@
+class Callback(object):
+    def __init__(self):
+        pass
+
+    def on_batch_begin(self):
+        pass
+
+    def on_batch_end(self):
+        pass
+
+    def on_epoch_begin(self):
+        pass
+
+    def on_epoch_end(self):
+        pass
+
+    def on_predict_batch_begin(self):
```

Comment:
Let's only do training-related methods and remove `predict` and `test` related methods.

Reply:
Removed in next commmit.